### DISCUSSION TITLE
Enquiry about the application of Github Enterprise

### DISCUSSION THREAD

**Discussion by HKPU-SDIT** [community/community#136256] [GitHub Education]
### Select Topic Area

Question

### Body

Dear,

For the github education application, does it mean that this is represent all / entire university for the application?

Because I just want to create account for School of Design for the HK polytechnic University. But your form need to provide the application form and sigh the form.
<img width="668" alt="image" src="https://github.com/user-attachments/assets/e99c9785-5b61-4227-aa4f-fdf2d24ba10d">

Am I correct for the application?

Thanks.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @HKPU-SDIT , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
